eessi/testsuite/tests/apps/osu.py

```diff
@@ -0,0 +1,314 @@
+"""
+This module tests the binary 'osu' in available modules containing substring 'OSU-Micro-Benchmarks'.
+The basic application class is taken from the hpctestlib to which extra features are added.
+"""
+import reframe as rfm
+from hpctestlib.microbenchmarks.mpi.osu import osu_benchmark
+
+from eessi.testsuite import hooks, utils
+from eessi.testsuite.constants import CPU, SCALES, TAGS, DEVICE_TYPES,\
+        COMPUTE_UNIT, GPU, GPU_VENDOR, FEATURES, GPU_VENDORS, NVIDIA
```

Comment:
some constants are missing here.
in the other tests we just do the following to make our lives easier:
```
from eessi.testsuite.constants import *  # noqa
```

Reply:
Incorporated.